src/main/java/org/truffleruby/cext/CExtNodes.java

```diff
@@ -1035,6 +1038,47 @@ protected Object rbConstSet(RubyModule module, String name, Object value,
 
     }
 
+    @Primitive(name = "rb_gv_get")
+    @NodeChild(value = "name", type = RubyNode.class)
+    public abstract static class RbGvGetNode extends PrimitiveNode {
+
+        @Child WarnNode warnNode;
+
+        @CreateCast("name")
+        protected RubyNode coerceToString(RubyNode name) {
+            return ToJavaStringNode.create(name);
+        }
```

Comment:
It's a Ruby `String` coming in, but the globals storage needs a `j.l.String`. I followed the same pattern being used by `RbConstGetNode`, `RbConstGetFromNode`, and `RbConstGetFromNode`, but since we don't need to consider symbols, I'll switch to `ToJavaStringNode`.